clang/test/AST/ByteCode/constexpr-vectors.cpp

```diff
@@ -0,0 +1,88 @@
+// RUN: %clang_cc1 %s -triple x86_64-linux-gnu -fexperimental-new-constant-interpreter -Wno-uninitialized -std=c++14 -fsyntax-only -verify
```

Comment:
Why no RUN line for the current interpreter?

Reply:
Added.